ckanext/validation/tests/test_form.py

```diff
@@ -80,7 +80,7 @@ def test_resource_form_create(self, app):
 
         dataset = call_action("package_show", id=dataset["id"])
 
-        assert dataset["resources"][0]["schema"] == value
+        assert json.loads(dataset["resources"][0]["schema"]) == value
```

Comment:
This will give an unhelpful error message if the value is None. Can it be extracted to a helper function to do a null check and give a descriptive error?